src/System.Device.Gpio/Interop/Unix/Libc/Interop.ioctl.cs

```diff
@@ -105,4 +105,4 @@ internal struct spi_ioc_transfer
     public byte tx_nbits;
     public byte rx_nbits;
     public ushort pad;
-};
+}
```

Comment:
Looks like we actually have a convention in the dotnet repos of not having semicolons after structs. Do you mind removing the other ones where we didn't follow the convention in this same PR? I can see that at least in this file `i2c_rdwr_ioctl_data` also has one.

Reply:
I scanned through the solution and only see the `i2c_rdwr_ioctl_data`.  Let me know if you see others.